node_modules/oae-tenants/config/config.js

```diff
@@ -91,12 +91,5 @@ module.exports = {
         'elements': {
             'timezone': new Fields.List('Tenant Timezone', 'Tenant timezone', 'Etc/UTC', timezoneConfigValues, {'suppress': true})
         }
-    },
```

Comment:
Will this cause an issue with the data that's already in production? (probably not)